app/helpers/report_helper.rb

```diff
@@ -146,4 +146,122 @@ def reports_group_title
          :group_description => User.current_user.current_group.description}
     end
   end
+
+  def dashboard_widget_summary(widget, sb_data)
+    schedule = widget.miq_schedule
+    timezone = schedule&.run_at ? schedule.run_at[:tz] : session[:user_tz]
+    safe_join([
+                widget_basic_information(widget),
+                widget_status(widget, sb_data, timezone),
+                widget_sb_type_report(widget, sb_data),
+                widget_timer(schedule, timezone, sb_data),
+                widget_visibility(widget),
+              ])
+  end
+
+  private
+
+  def row_data(label, value, read_only = "")
+    {:cells => {:label => "#{read_only} #{_(label)}", :value => value}}
+  end
+
+  def widget_basic_information(widget)
+    read_only = widget.read_only ? '*' : ''
+    data = {:title => _('Basic Information'), :mode => "miq_widget_basic_info"}
+    data[:rows] = [
+      row_data('Title', widget.title, read_only),
+      row_data('Descrption', widget.description, read_only),
+      row_data('Active', widget.enabled),
+      row_data('Default', widget.read_only)
+    ]
+    miq_structured_list(data)
+  end
+
+  def widget_status(widget, sb_data, timezone)
+    unless sb_data[:wtype] == "m"
+      data = {:title => _('Status'), :mode => "miq_widget_status_info"}
+      glyphicon = widget.decorate.try(:secondary_icon)
+      current_status = glyphicon ? [{:icon => glyphicon, :value => widget.status}] : widget.status
+      data[:rows] = [
+        row_data('Current Status', current_status),
+        row_data('Last Run Time', h(widget.last_run_on ? format_timezone(widget.last_run_on, timezone, "view") : "")),
+        row_data('Message', h(widget.status_message))
+      ]
+      miq_structured_list(data)
+    end
```

Comment:
Prefer flipping the first check to be a guard clause:

```suggestion
    return if sb_data[:wtype] == "m"

    data = {:title => _('Status'), :mode => "miq_widget_status_info"}
    glyphicon = widget.decorate.try(:secondary_icon)
    current_status = glyphicon ? [{:icon => glyphicon, :value => widget.status}] : widget.status
    data[:rows] = [
      row_data('Current Status', current_status),
      row_data('Last Run Time', h(widget.last_run_on ? format_timezone(widget.last_run_on, timezone, "view") : "")),
      row_data('Message', h(widget.status_message))
    ]
    miq_structured_list(data)
```